compose-builder/add-secure-redis-messagebus.yml

```diff
@@ -0,0 +1,32 @@
+# /*******************************************************************************
+#  * Copyright 2021 Intel Corporation.
+#  *
+#  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+#  * in compliance with the License. You may obtain a copy of the License at
+#  *
+#  * http://www.apache.org/licenses/LICENSE-2.0
+#  *
+#  * Unless required by applicable law or agreed to in writing, software distributed under the License
+#  * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  * or implied. See the License for the specific language governing permissions and limitations under
+#  * the License.
+#  *
+#  *******************************************************************************/
+
+version: '3.7'
+
+volumes:
+  kuiper-config:
+
+services:
+  secretstore-setup:
+    volumes:
+      - kuiper-config:/tmp/kuiper:z
+    environment:
+      SECUREMESSAGEBUS_TYPE: redis
+
+  rulesengine:
+    volumes:
+      - kuiper-config:/kuiper/etc/sources:z
+    depends_on:
+      - secretstore-setup
```

Comment:
Should `rulesengine` running in secure mode wait for redis database?  we should use secuirty-bootstrapper to guard the starting sequence instead of relying only on `depends_on`.

Reply:
I consider that but since the service has to wait on the volume mount, it looks to be starting in sequence fine. Also kuiper doesn't connect until a rule is created. I will test again w/o removing volume so Kuiper creates saved rule and connects on start-up, but I think we are ok.

Reply:
@jim-wang-intel , Yup need to have it wait till Redis gate is open.  Looks like it tried to use the previous password in the config file... :-(
```
time="2021-06-03 16:09:14" level=info msg="closing rule anyevent for error: WRONGPASS invalid username-password pair or user is disabled." file="server/ruleManager.go:97"
```

Reply:
yea, we can add that waiting into the entrypoint script

Reply:
Added